src/components/search/PageSearchModal/index.js

```diff
@@ -111,14 +117,21 @@ module.exports = React.createClass({
     } else {
       this.setState(this.getInitialState());
     }
+
+    this.props.resizeCallback();
   },
 
-  selectHandler: function(page, event) {
+  onClose: function() {
     this.props.onClose();
+    this.props.resizeCallback();
+  },
+
+  selectHandler: function(event, result) {
+    this.onClose();
 
-    if (this.props.onSelect) {
+    if (this.props.action == 'custom' && this.props.onSelect) {
```

Comment:
Very petty, and I doubt it would matter unless someone was using the `String()` constructor for some reason. But is there a reason for not using `===` instead of `==`?